pack/ramp-community.yml

```diff
@@ -0,0 +1,25 @@
+display_name: RediSearch 2
```

Comment:
Let's align the file name, e.g., `ramp-oss-cluster`

Reply:
OK. Done.